### PR TITLE
Add Coveralls

### PR DESCRIPTION
**Pull request by SillyInventor**
What it sounds like.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -20,11 +20,15 @@ install:
...
 
 before_script:
   - psql -c "CREATE DATABASE travisci;" -U postgres
 
 script:
   - python manage.py migrate
-  - python manage.py test rpi_csdt_community
-  - python manage.py test project_share
\ No newline at end of file
+  - coverage run --source=rpi_csdt_community,meetup manage.py test
```
Comment (chuck211991) [inline .travis.yml]: What is "meetup"?

File: .travis.yml [hunk 2]
```diff
@@ -20,11 +20,15 @@ install:
...
   - sudo pip install -r libraries-heroku.txt
+  - sudo pip install coveralls
 
 before_script:
   - psql -c "CREATE DATABASE travisci;" -U postgres
 
 script:
   - python manage.py migrate
-  - python manage.py test rpi_csdt_community
-  - python manage.py test project_share
```
Comment (chuck211991) [inline .travis.yml]: Why was this removed, but not added back in to the new script?

File: .travis.yml (hunk 1 above)
Comment (SillyInventor) [inline .travis.yml]: Meetup was a mistake I made (I thought it was part of the syntax of calls to coverage)

File: .travis.yml (hunk 2 above)
Comment (SillyInventor) [inline .travis.yml]: Calling the general test should call both.

**Comment by SillyInventor** [comment]
@chuck211991 also, remember, we're planning on making Ryan review this to get him into the spirit of things... so, you know, don't approve.
